include/hardware.h

```diff
@@ -80,6 +83,9 @@ void PS1DAC_NotifyUnlockMixer();
 void GUS_NotifyLockMixer();
 void GUS_NotifyUnlockMixer();
 
+void DISKNOISE_NotifyLockMixer();
+void DISKNOISE_NotifyUnlockMixer();
```

Comment:
These aren't needed. I don't see that you actually wrote these functions anyway so just delete these two lines.

The devices that have these functions are special in that they generate their audio on the main thread and need to be notified when the mixer is being locked so they can stop a queue and avoid a deadlock.